src/Core/Manifest/VersionProvider.php

```diff
@@ -180,16 +180,23 @@ public function getModules()
     /**
      * Tries to obtain version number from composer.lock if it exists
      *
-     * @param array $modules
-     * @return array
+     * @param array<int,string> $modules
```

Comment:
```suggestion
     * @param array<string> $modules
```
The type of the keys doesn't matter here. You can use `array-key` if you want to be really explicit.